sublime_jedi/py_select.py

```diff
@@ -0,0 +1,44 @@
+import sublime, sublime_plugin
+from .utils import PythonCommandMixin, is_python_scope
+from .settings import get_plugin_settings
+
+
+class PySelectEventListener(sublime_plugin.EventListener):
+
+    def show_py(self, view) -> None:
+        '''
+        Show the Python Interpreter used by SublimeJEDI on statusbar
+        '''
+        if is_python_scope(view, view.sel()[0].begin()):
+            view.window().status_message('   Py: {}'.format(get_plugin_settings().get('python_interpreter')))
+
+    def on_load(self, view) -> None:
+        self.show_py(view)
+
+    def on_activated(self, view) -> None:
+        self.show_py(view)
+
+
+class ListAvaliablePython(sublime_plugin.ListInputHandler):
+    def __init__(self, view):
+        self.view = view
+
+    def list_items(self):
+        return get_plugin_settings().get('python_interpreters')

```

Comment:
I was wondering what if `get_plugin_settings().get('python_interpreters')` returns `None`?
ST3 will show an error?

Reply:
ST3 will report a TypeError in the Console.
I have added settings check code to ensure that SublimeJEDI will not be affected when there are no 'python_interpreter' or 'python_interpreters' setting in sublime_jedi.sublime-settings.